core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -76,6 +76,11 @@ object AclCommand {
     finally CoreUtils.swallow(authZ.close())
   }
 
+  private def description(opts: AclCommandOptions): String = {
+    withAuthorizer(opts) { authorizer => return authorizer.description()}
```

Comment:
This syntax is a bit unusual. Especially using a return and also the "" at the end of the method. I think this could be cleaned up by changing the signature of `withAuthorizer` to return the type of the passed function.

From:
```scala
def withAuthorizer(opts: AclCommandOptions)(f: Authorizer => Unit) {
```

To:
```
def withAuthorizer[T](opts: AclCommandOptions)(f: Authorizer => T): T = {
```